travis/install_custom_pg

```diff
@@ -58,7 +58,10 @@ cd postgresql
 
 make -j "${mjobs}" -s all
 make -j "${mjobs}" -s -C src/test/isolation
+make -s -C contrib/pg_stat_statements
```

Comment:
Maybe add `-j "${mjobs}"` too for consistency and slight speed-up

Reply:
ok